hugegraph-api/src/main/java/com/baidu/hugegraph/config/ServerOptions.java

```diff
@@ -51,9 +51,20 @@ public static synchronized ServerOptions instance() {
     public static final ConfigOption<Integer> MAX_WORKER_THREADS =
             new ConfigOption<>(
                     "restserver.max_worker_threads",
-                    "The maximum worker threads of rest server.",
+                    "The maxmium worker threads of rest server.",
                     positiveInt(),
-                    2 * Runtime.getRuntime().availableProcessors());
+                    2 * Runtime.getRuntime().availableProcessors()
+            );
+
+    public static final ConfigOption<Long> MIN_FREE_MEMORY =
+            new ConfigOption<>(
+                    "restserver.min_free_memory",
+                    "The minmium free memory(MB) of rest server, requests " +
+                    "will be rejected when the available memory of system " +
+                    "is lower than this value.",
+                    positiveInt(),
+                    128L
```

Comment:
is it a suitable default value?

Reply:
Choosed 512M, which is also the minimum memory required to start the service.